packages/terra-hookshot/src/Hookshot.jsx

```diff
@@ -0,0 +1,335 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Portal from 'react-portal';
+import HookshotContent from './HookshotContent';
+import HookshotUtils from './_HookshotUtils';
+
+const ATTACHMENT_POSITIONS = [
+  'top left',
+  'top center',
+  'top right',
+  'middle left',
+  'middle center',
+  'middle right',
+  'bottom left',
+  'bottom center',
+  'bottom right',
+];
+
+const ATTACHMENT_BEHAVIORS = [
+  'auto',
+  'flip',
+  'none',
+];
+
+const propTypes = {
+  /**
+   * How the content should be positioned when the primary attachment is not available.
+   * Valid values: 'auto', 'flip', and 'none'.
```

Comment:
Is the behavior of `auto`, `flip`, `none` defined somewhere. How do consumers know what these do? The answer could just be we have examples of the behavior to highlight it, but I didn't see any written doc on what these do.